x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_table_filters.ts

```diff
@@ -21,7 +21,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const to = '2024-01-01T12:00:00.000Z';
 
-  // FLAKY: https://github.com/elastic/kibana/issues/178652
   describe.skip('Dataset quality table filters', () => {
```

Comment:
should this be unskipped as well?

Reply:
Good catch, I missed the keyword completely